getExclusions.m

```diff
@@ -11,49 +10,52 @@
                 tcs.Move.(types{1}));
 nUnits = tmp(:, 1);                             % how many units per area                    
 
-% preallocate
-for ia = 1:nAreas
-   OK.isVisual{ia} = zeros(nTypes, nUnits(ia));   
-   OK.anyVisual{ia} = zeros(1, nUnits(ia));   
-   OK.DSI{ia} = zeros(nTypes, nUnits(ia));
-end
-
-
 % Must respond more to move than blank throughout experiment
 %   in at least one of the 12 directions
 
 % parameters 
 pThresh = 0.05;
-nBins = 4;
-
+trialStep = 10; %test 10 trials at a time for visual responsiveness (move > blank)
 
 for ia = 1:nAreas   
 for it = 1:nTypes
-    overThresh = zeros(nUnits(ia), nDirs);
+    p = nan(nUnits(ia), nBins, nDirs);
 for id = 1:nDirs
     tmpMV = tcs.Move.(types{it}){ia,id};
     tmpBK = tcs.Blank.(types{it}){ia,id};
     
     nTrials = size(tmpBK, 2);
-    trialStep = nTrials/nBins;
+    nBins = ceil(nTrials/trialStep);
     binStarts = 1:trialStep:nTrials;
```

Comment:
Some comments might help here. I'm confused whether the sliding-over-trials analysis is intended to:
1) examine significance with 1-trial resolution, but in a window of nBins trials
2) examine significance with nBins resolution, in a window of nBins trials. 
I think option 1 is desirable, but you've implemented option 2 (possibly).